turbot/__init__.py

```diff
@@ -811,23 +852,28 @@ def predict_command(self, channel, author, params):
         yours = prices[(prices.author == target_id) & (prices.timestamp > past)]
         yours = yours.sort_values(by=["timestamp"])
 
+        # convert all timestamps to the target user's timezone
+        target_timezone = self.get_user_timezone(target_id)
+        prices["timestamp"] = pd.to_datetime(prices["timestamp"])  # FIXME: Why needed?
```

Comment:
I don't quite understand why this line is needed but I think I know how I might be able to fix it. I'll address it in another PR in the future.

Reply:
Fixed!